### PR TITLE
Deploy v1.3

### PR DESCRIPTION
**Pull request by julkascript**

### CODE REVIEW

File: frontend/src/translations/en/account.json [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+      "hint": "You will be logged out after the change."
+    },
+    "defaultShipmentAddress": {
+      "title": "Default shipment address",
+      "description": "Enter the default shipment address that will get pre-filled.",
+      "hint": "You can always change the address per purchase in the checkout section."
+    },
+    "defaultCurrency": {
+      "title": "Default currency",
+      "hint": "Changing the currency does not affect the price. The new rates will be applied. You will be logged out after changing your default currency."
```
Comment (RyotaMitaraiWeb) [inline frontend/src/translations/en/account.json:54]: this one's my fault, but we should probably remove the last sentence before we deploy (same for the Bulgarian equivalent), it's a leftover from a previous implementation and I forgot to update the translations

**Review by RyotaMitaraiWeb — APPROVED** [review]
Looks good to me overall

**Comment by julkascript** [comment]
@julkascript add v1.3 changelog

**Review by dinocom33 — APPROVED** [review]
Looks good to me.

**Comment by julkascript** [comment]
> @julkascript add v1.3 changelog

Done.

**Pull request merged**